### ISSUE TITLE
Swiper Nested Keyboard Navigation Does Not Work with Virtual Slides

### ISSUE THREAD

**Issue by kukicado** [nolimits4web/swiper#6378]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://codesandbox.io/p/sandbox/swiper-react-nested-forked-cc3z5o?file=%2Fsrc%2FApp.jsx&selection=%5B%7B%22endColumn%22%3A50%2C%22endLineNumber%22%3A38%2C%22startColumn%22%3A50%2C%22startLineNumber%22%3A38%7D%5D

### Bug description

When you have Nested Swipers in going in different directions (for example Parent Swiper - Vertical, Child Swiper - Horizontal) and have Keyboard navigation enabled on both, only the parent keyboard navigation works, so in the above example you can use up and down keys to move vertically, but not horizontally via the keyboard.

If you disable Virtual navigation on both, then the keyboard commands work just fine for both parent and child. Attached codesandbox has parent navigation horizontal, and child swiper veritcal, and shows the same issue, which again is resolved if your remove virtual slides.

### Expected Behavior

You should be able to use keyboard navigation for nested Swiper's with the Virtual module. 

### Actual Behavior

Only the parent keyboard events are captured, so whichever direction the parent is set, you can only use the keyboard for that direction.

### Swiper version

9.0.3

### Platform/Target and Browser Versions

Windows

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by kukicado** [comment]
An interesting workaround that does work is breaking up the Swiper with an additional div. 

So if the tree looks like:

```
<Swiper>
  <SwiperSlide>
      <div>something here</div>
      <Swiper>
          <SwiperSlide></SwiperSlide>
      </Swiper>
   </SwiperSlide>
</Swiper>
```

Then the navigation seems to work ok in both directions.

**Comment by nolimits4web** [comment]
Each slide must have `virtualIndex` property

**Comment by kukicado** [comment]
That did it. Thank you!
